dev/svelte/ColorChange.svelte

```diff
@@ -2,7 +2,7 @@
   export let scatterplot;
 
   let value = 'category10';
-  const schemes = ['okabe', 'category10', 'dark2', 'pastel2'];
+  const schemes = ['okabe', 'category10', 'dark2', 'pastel2', 'observable10'];
```

Comment:
Adding one new colorscheme while we're in here.

Reply:
can we call it nomic10

Reply:
Nomic 10 will be even awesomer